opa/security/opa_security.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="module_opa_category" model="ir.module.category">
```

Comment:
Why are you adding this?

Reply:
To place all `res.groups` related to Open Academy on that category.